tools/licenses/lib/main.dart

```diff
@@ -2099,10 +2099,10 @@ class _RepositoryFlutterDirectory extends _RepositoryDirectory {
 /// A specialized crawler for "github.com/flutter/engine/lib" directory.
 ///
 /// It includes everything except build tools, test build artifacts, and test code.
-_RelativePathBlacklistRepositoryDirectory _createLibDirectoryRoot(fs.Directory entry, _RepositoryDirectory parent) {
-  return _RelativePathBlacklistRepositoryDirectory(
+_RelativePathBlocklistRepositoryDirectory _createLibDirectoryRoot(fs.Directory entry, _RepositoryDirectory parent) {
```

Comment:
IIRC Dart went with "DenyList" do we want to be consistent with their terminology?

Reply:
Changed.